jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/graphql_dsl/DslGraphqlSampler.java

```diff
@@ -0,0 +1,57 @@
+package us.abstracta.jmeter.javadsl.graphql_dsl;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.http.entity.ContentType;
+import org.apache.jmeter.protocol.http.config.GraphQLRequestParams;
+import org.apache.jmeter.protocol.http.util.GraphQLRequestParamUtils;
+import us.abstracta.jmeter.javadsl.http.DslHttpSampler;
+
+import java.util.HashMap;
+
+public class DslGraphqlSampler extends DslHttpSampler {
+
+    private final GraphQLRequestParams params = new GraphQLRequestParams();
+    private final HashMap<String, Object> variables = new HashMap<>();
+
+    public DslGraphqlSampler(String name, String url) {
+        super(name, url);
+        this.contentType(ContentType.APPLICATION_JSON);
```

Comment:
In my opinion we should set it inside, as default, because it is as standard graphql option. If fir some reason user is needed to change it we should provide a method to do so.

Reply:
Yes, I agree, is better to avoid common pitfall and reduce code for most of the scenarios and request additional code for corner cases, than the opposite.